ehrql/codes.py

```diff
@@ -100,6 +100,12 @@ class SNOMEDCTCode(BaseCode):
     regex = re.compile(r"[1-9][0-9]{5,17}")
 
 
+class SNOMEDYCode(BaseCode):
```

Comment:
With the caveat the almost anything I say about clinical coding systems could turn out to be wrong, I'm _pretty_ sure that there's no such thing as SNOMED-Y Codes.

SNOMED codes are always integers (they're designed to fit into 64 bit int columns).

I think Y-codes are _something_ like a TPP-specific subset of CTV3 in which case we can use the existing `CTV3Code` type for this – but we'd need to confirm that.

Reply:
Ah! I have probably misunderstood/misremembered. There are such things as "Y" codes, and they are TPP-specific, but they may well be CTV3 codes. I know @inglesp knows more. Could he confirm?